### ISSUE TITLE
The icons on the Cluster page are cut off on the right side

### ISSUE THREAD

**Issue by mmoulian** [weaveworks/weave-gitops#3908]
The icons on the Cluster page are cut off on the right side. In addition to fixing this small design detail, my proposal is to change all the icons from grey to their own corporate colour. If we do this, the user would be able to easily recognize all kinds of Clusters.

**Image of the current demo:**
![Captura de Pantalla 2023-08-02 a las 16.56.08.png](https://images.zenhubusercontent.com/64ad7261e8ff8333028723ac/4eaa1c9c-a971-47e1-9762-140c906b4416)

**Image of the Figma design:**
![Captura de Pantalla 2023-08-02 a las 17.08.37.png](https://images.zenhubusercontent.com/64ad7261e8ff8333028723ac/b3f05da2-57a6-4274-92a4-e7785ed41dc3)

See the design here:
https://www.figma.com/file/IVHnM9iyeFWpd11evtY8ux/Weave-GitOps?type=design&node-id=17914%3A199097&mode=design&t=Oy9FiwH9kVdhRT0X-1

You can find all types of cluster icons in this folder at 32px size:
https://drive.google.com/drive/folders/1vXJQ_U5TkM_Hw6wrQ5IBSm3yaC815-sj?usp=sharing

**Comment by joshri** [comment]
Hey Manuela! 

Can you direct me to who you got the list of cluster types from? I need their help to get the exact `type` strings for all the new kinds. We're also missing the `Kind` icon for Docker clusters, and we have an icon for `OtherOnprem` - not sure what that case is addressing specifically

**Comment by mmoulian** [comment]
Hey!! I came from this page : [ weave-gitops-enterprise/ui-cra/src/utils/icons](url). You sent it to me a while ago and I took a screenshot to work on Illustrastor. I marked with a rectangle the icons that I couldn't find. The OtherOnprem was in the unknown icons. Investigating now I realized that they are the local machines. It is right? Anyway I couldn't find the icon. Is there a standard? or Can we create one? I added to the folder two Docker icons (regular and simple)
<img width="1156" alt="Captura de Pantalla 2023-08-09 a las 15 46 19" src="https://github.com/weaveworks/weave-gitops/assets/95616291/35f405ff-a066-4bdf-ae46-5a721d230b9e">
